test/suites/ipv6/suite_test.go

```diff
@@ -94,4 +100,58 @@ var _ = Describe("IPv6", func() {
 		})
 		Expect(internalIPv6Addrs).To(HaveLen(1))
 	})
+	It("should provision a static IPv6 prefix with node launch and set IPv6 as primary in the primary network interface", func() {
+		clusterDNSAddr := env.ExpectIPv6ClusterDNS()
+		nodeClass.Spec.Kubelet = &v1.KubeletConfiguration{ClusterDNS: []string{clusterDNSAddr}}
+		Expect(disableVPCCNIProvisioning(true)).ToNot(HaveOccurred())
```

Comment:
This won't be undone when you are done with the test if you don't reset it back to its prior state. I wonder if it's worth going through the process of disabling and re-enabling this feature on the CNI for just checking that we passed a setting in the launch?